js/components/contextMenu.js

```diff
@@ -157,6 +157,11 @@ class ContextMenuItem extends ImmutableComponent {
             style={iconStyle}></span>
           : null
       }
+      {
+        this.hasSubmenu
+        ? <span className='submenuIndicator contextMenuIcon bookmarkIcon fa fa-folder-o' style={iconStyle} />
```

Comment:
The word bookmark shouldn't be in this file since it's a general context menu file.